test/main_test.py

```diff
@@ -1,11 +1,22 @@
 import pytest
 import sqlite3
 from typing import Union
-from .context import main
-from .utils import populate_db
+from test.context import main, main_menu
+from test.utils import populate_db
+
+"""
+Beginning of tests for sprint #1
+"""
 
 SAMPLE_USERNAME: str = "User1"
 SAMPLE_PASSWORD: str = "$Aample123"
+<<<<<<< Updated upstream
+=======
```

Comment:
Git metadata should be removed. I don't think the program will run with this in it.